### PR TITLE
align search and error messages

### PR DESCRIPTION
**Pull request by mertkahyaoglu**
<b>Before:</b>
screen shot 2015-11-26 at 23 11 33
screen shot 2015-11-26 at 23 21 20

<b>After:</b>
screen shot 2015-11-26 at 23 23 28

### CODE REVIEW

File: styles/settings-view.less [hunk 1]
```diff
@@ -570,13 +570,12 @@
       }
     }
 
-    .search-message,
-    .featured-message {
-      margin: @component-padding @component-padding 0 @component-padding;
+    .search-message {
```
Comment (lee-dohm) [inline styles/settings-view.less]: What about `.featured-message`?

Comment (simurai) [inline styles/settings-view.less]: Tried to find if `.featured-message` gets used, but doesn't seem so. Probably a left-over. I think it's ok to remove it.

**Comment by simurai** [comment]
Looks good. @mertkahyaoglu thanks.

**Pull request merged**